__init__.py

```diff
@@ -732,12 +746,19 @@ def _speak_expired_timer(self, expired_timers):
         """Announce the expiration of any timers not already announced.
 
         This occurs every two seconds, so only announce one expired timer per pass.
+        Pause the expiration check so the expired timer is not beeping while the
+        expiration announcement is being spoken.
+
+        On the Mark I, pause the display of any active timers so that the mouth can
+        do the "talking".
         """
         for timer in expired_timers:
             if not timer.expiration_announced:
                 dialog = TimerDialog(timer, self.lang)
                 dialog.build_expiration_announcement_dialog(len(self.active_timers))
-                self._pause_scheduled_events()
+                self._stop_expiration_check()
+                if self.platform == MARK_I:
+                    self._stop_display_update()
                 time.sleep(1)  # give the scheduled event a second to clear
                 self.speak_dialog(dialog.name, dialog.data)
```

Comment:
I think this needs a wait or you will set the completed indicator before it has completed

Reply:
Fixed.